LearningMissionSimulation/PlaygroundGavril.cs

```diff
@@ -243,6 +255,30 @@ public void RegisterStudentsForClasses()
 
             ReportSummary(actionName: action, itemCount: classroomItemListCount);
             ReportFooter(actionName: action);
+        }
+
+        private void UpdateStudentList(Classroom classroom)
+        {
+            int itemListCount = AttributeGenerator.random.Next(0, classroom.MaximumCapacity - classroom.ItemList.Count + 1);
+            int i = 0;
+            while (i < itemListCount)
+            {
+                foreach (var student in activeStudentList)
+                {
+                    if (!student.CompletedModuleList.Contains(classroom.Module))
```

Comment:
You have to check also if the student is already registered for the class. We don't want to register the same student multiple times for the same class.

Reply:
Done

Reply:
DONE